k-distribution/include/kore/prelude.kore

```diff
@@ -47,4 +47,8 @@ module K
   alias weakAlwaysFinally{A}(A) : A
   where weakAlwaysFinally{A}(X:A) := X:A []
 
+  // Definitions for CTL aliases
+  alias allPathGlobally{A}(A) : A
+  where allPathGlobally{A}(X:A) := X:A []
```

Comment:
I remember you said this is not sound, which obviously it's not since it's not actually doing anything here. Can we get a TODO then?

Reply:
Added a same as weakExistsFinally.